tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_training_cuda10_2

```diff
@@ -35,6 +35,11 @@ COPY build_scripts/install-entrypoint.sh \
      build_scripts/build_utils.sh \
      /build_scripts/
 
+# Updating the CUDA Linux GPG Repository Key ( https://developer.nvidia.com/blog/updating-the-cuda-linux-gpg-repository-key/ )
```

Comment:
Why inference cuda 10.2 pipeline doesn't have the problem? If the base image is broken, inference pipelines should have problems too.

Reply:
the link shows fixed pipeline, add broken pipeline now.

Reply:
Only training cuda10.2 python package pipeline using nvidia/cuda:10.2-cudnn7-devel-centos7. So only training pipeline broken.

Reply:
Inference uses "nvidia/cuda:10.2-cudnn8-devel-centos7", are you saying cudnn7 is bad but cudnn8 is good?